src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -7,17 +7,34 @@ import { directions } from '../../styles/constants';
 
 const ALIGMENT_TYPES = [directions.LEFT, directions.CENTER, directions.RIGHT];
 
-const marginStyles = ({ theme, noMargin }) =>
-  !noMargin &&
-  css`
-    li:not(:last-of-type) {
+const getInlineStyles = theme => css`
+  display: flex;
+  flex-wrap: nowrap;
+
+  > li:not(:last-of-type) {
+    margin-bottom: 0;
+    margin-right: ${theme.spacings.mega};
+  }
+`;
+
+const baseStyles = ({ theme }) => css`
+  label: button-group;
+  list-style-type: none;
+  overflow: hidden;
```

Comment:
Also, the `overflow: hidden` crops of the bottom border of the wrapped buttons.

Reply:
Where?

<img width="685" alt="screen shot 2018-07-24 at 13 43 15" src="https://user-images.githubusercontent.com/265947/43135996-8f510612-8f47-11e8-9485-399638081a14.png">

Reply:
But I'll remove the overflow completely. Don't know why it's there to begin with.

Reply:
This is what it looked like on the website:

![screenshot 2018-07-24 13 46 22](https://user-images.githubusercontent.com/11017722/43136126-f974baac-8f47-11e8-9f66-74bce0b081de.png)